scala-mode2.el

```diff
@@ -169,5 +170,11 @@ When started, runs `scala-mode-hook'.
                '("\\.\\(scala\\|sbt\\)\\'" . scala-mode))
   (modify-coding-system-alist 'file "\\.\\(scala\\|sbt\\)\\'" 'utf-8))
 
+(defvar scala--prettify-symbols-alist scala-mode-pretty-all-alist)
+(defun scala-add-pretty ()
+  (setq prettify-symbols-alist scala--prettify-symbols-alist))
```

Comment:
The difference is that if the user had set something already in prettify-symbols-alist, then this using `setq` would override that